client/state/selectors/is-jetpack-module-active.js

```diff
@@ -14,5 +16,15 @@ import { get } from 'lodash';
  * @returns {?boolean}            Whether the module is active
  */
 export default function isJetpackModuleActive( state, siteId, moduleSlug ) {
+	if ( moduleSlug === 'photon' || moduleSlug === 'photon-cdn' ) {
+		// When site is atomic and private, we filter out photon from active modules list.
+		// This isn't actually changing any stored preferences, which means photon is going to
+		// keep working once privacy is disabled.
+		const siteIsAtomicPrivate =
+			isSiteAutomatedTransfer( state, siteId ) && isPrivateSite( state, siteId );
```

Comment:
Should this become a `isSiteAtomicPrivate` selector since we're calculating it twice in this PR (& possible elsewhere)?

Reply:
Good point, it crossed my mind too! I added a new card to the board to clean up all places like that, we have more than just this PR.